.github/workflows/python-release.yaml

```diff
@@ -0,0 +1,215 @@
+name: Python Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+    branches-ignore:
+      - '*'
```

Comment:
I think this should not run at all on pushes, right? Otherwise it will run twice, once when the push happens and again after `Python Test` finished?

Reply:
Oh, right, but we don't want to release for every `Python Test` run, only runs for tags :thinking: 

Maybe we need one more condition for the `if` in the starting jobs, like `startsWith(github.ref, 'refs/tags/v')`.

Reply:
I guess this workflow will need some testing before merging. Maybe you can make the publishing just a mock (like a script with `true`) and try it in your fork with PRs, pushing and tagging to make sure it works as expected.

Reply:
After testing it on my fork. There seems to be a problem with the triggers. As we trigger on both `push` and `workflow_run`, the workflow is called twice, when pushing a tag (as the tests also run). To circumvent this, I thought I could listen to only `workflow_run`, but then I cannot filter for the tags (https://github.com/actions/starter-workflows/issues/1137). I think the best solution is actually to have it all in one file... What do you think, @llucax ?

Reply:
Yeah, this is what I imagined. I was suggesting to add an `if` condition for every job in this workflow with `if: startsWith(github.ref, 'refs/tags/v')` to make sure it only runs on tags, but it is true that if this workflow will trigger for every push and PR and steps will be just skipped unless it is a tag, then we could as well merge both workflows. I guess the only advantage of having them separated could be more composability if we use reusable workflows at some point? But I'm not even sure...

So I would say feel free to merge if that's simpler for now.

In the future I would like to reverse the triggering of the release workflow, so releases are triggered manually instead via tagging, so the actual tagging is done by the release workflow. This can remove a lot of work that's currently done manually, like the tagging itself and the clearing of the release notes.

If we do this, it makes sense to split again.